lametro/events.py

```diff
@@ -342,23 +344,27 @@ def scrape(self, window=None):
                     ]
 
                 # Historically, the Legistar system has duplicated the EventItemAgendaSequence,
-                # resulting in data inaccuracies. The scrape should fail in such cases, until Metro
-                # cleans the data.
-                item_agenda_sequences = [
-                    item["extras"]["item_agenda_sequence"] for item in e.agenda
-                ]
-                if len(item_agenda_sequences) != len(set(item_agenda_sequences)):
-                    error_msg = "An agenda has duplicate agenda items on the Legistar grid: \
-                        {event_name} on {event_date} ({legistar_api_url}). \
-                        Contact Metro, and ask them to remove the duplicate EventItemAgendaSequence."
-
-                    raise ValueError(
-                        error_msg.format(
-                            event_name=e.name,
-                            event_date=e.start_date.strftime("%B %d, %Y"),
-                            legistar_api_url=legistar_api_url,
+                # resulting in data inaccuracies. In such cases, we'll log the event
+                # and notify Metro that their data needs to be cleaned (rather than failing).
+                try:
```

Comment:
Will this also skip scraping the event with the funky agenda?

Reply:
So the effect is that the yielded event won't have an associated agenda, i.e., the event will show up on the board agendas site sans agenda, until the duplicate is fixed? I think that should be ok, but we should let Metro know.